Bump d.py version to 2.6.2

Jackenmen opened the pull request:
### Description of the changes

Not much to do really, just a few things around component stuff.

### Have the changes in this PR been tested?

No

Jackenmen committed:
Bump d.py version to 2.6.2

Jackenmen committed:
Support new components in Filter cog

Jackenmen committed:
Update deprecated use of TextInput.label

Jackenmen committed:
Revert "Update deprecated use of TextInput.label"

This reverts commit efe1fbc5fc1af15cf0d5ca1c1d8474f3925b76d3.

Jackenmen committed:
Fix issues with optional fields

Kowlin reviewed (dismissed):
Probably works, right?

Jackenmen committed:
Stop checking placeholder field on the UnfurledMediaItem

It's just a thumbhash of the media, not user-provided text:
https://docs.discord.food/resources/components#unfurled-media-item-object-structure
